### ISSUE TITLE
Broken page in docs

### ISSUE THREAD

**Issue by jacg** [jacg/nain4#187]
Something is amiss [here](https://jacg.github.io/nain4/how-to/build-a-minimal-nain4-app.html).

Most obviously, the box after

> With this, our setup is ready, and we can start the simulation with

is completely empty.

Also, there is a box containing nothing besides

>   // Important! physics list has to be set before the generator!

where it doesn't make sense.

It seems that the positioning of the anchors in the source code got mangled at some stage.

**Comment by gonzaponte** [comment]
We've probably already had this conversation, but I don't remember the conclusion.
I think there is a conflict here. Originally, this document was written to showcase the _minimal_ `n4` app, but for that, we used a template that quickly grew to be a more complete example. The idea was that anyone could start a project with this template and have a fair amount of the functionality already exposed.
Before fixing this, I would like to discuss which approach to follow:

1. Create a really minimal app/template and focus this page on that (simpler) version. Then, add other pages documenting expansions to that minimal version.
2. Rewrite the page to account for this more complete example.

I believe we want to have both: a how-to explaining a somewhat complete example in detail, at the expense of feeling heavier on the reader; and a guide that builds something complex from scratch. Currently, this page is classified as a how-to, so it should be the former, but then it's not really a _minimal_ app, is it?

I can't promise any quick progress, but I might have a few hours to spare in the coming months that I could dedicate to this project.

**Comment by jacg** [comment]
I think we went back and forth on this. The fundamental problem is lack of manpower. We want all of

+ minimal app
+ bunch of HOWTO steps demonstrating various expansions
+ a more complex app that people can adapt to their needs

But all these for-human-consumption code+docs combinations require a lot of effort to make as legible and helpful as possible and cannot really be tested automatically.

We were under huge time pressure when we wrote the app+docs, which explains the confusion in the role they play. Given that we have approximately zero manpower on this project at present, there is not much we can do to fix it.

TLDR: yes we want both, but you can't have everything you want, especially if you don't have resources. So, do what you feel is realistically achievable in whatever time you can devote to this.

Not very helpful. Sorry.
